pkg/asset/cluster/tfvars/tfvars.go

```diff
@@ -219,6 +219,13 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		return errors.Errorf("master slice cannot be empty")
 	}
 
+	numWorkers := int64(0)
+	for _, worker := range installConfig.Config.Compute {
+		if worker.Replicas != nil {
+			numWorkers += *worker.Replicas
+		}
```

Comment:
There is a nice function for this in `k8s.io/utils/ptr`:
```suggestion
		numWorkers += ptr.Deref(worker.Replicas, 0)
```

Reply:
Done.